server/server.go

```diff
@@ -24,14 +24,16 @@ import (
 // other components that the the finality gadget server needs to run.
 type Server struct {
 	proto.UnimplementedFinalityGadgetServer
-	fg  finalitygadget.IFinalityGadget
-	cfg *config.Config
-	db  db.IDatabaseHandler
-
-	logger *zap.Logger
 
+	grpcServer  *grpc.Server
+	httpServer  *http.Server
```

Comment:
Do we need both servers or keeping one of them is good enough?

Reply:
we need both, grpc server is used for op-node and http is for finality explorer